app/models/contact_topic_answer.rb

```diff
@@ -0,0 +1,31 @@
+class ContactTopicAnswer < ApplicationRecord
+  belongs_to :case_contact
+  belongs_to :contact_topic
+
+  validates :selected, inclusion: [true, false]
+
+  default_scope { joins(:contact_topic).order("contact_topics.question") }
+end
+
+# == Schema Information
+#
+# Table name: contact_topic_answers
+#
+#  id               :bigint           not null, primary key
+#  selected         :boolean          default(FALSE), not null
```

Comment:
Is there functionality of `selected` beyond it's effect on the Notes page? Not saying it shouldn't be a database column, I'm just curious what you envision.

Reply:
Overall, I totally agree with your points. This is how the Casa folks want it. (That might potentially change when they actually get their hands on it)

Brandon says this how the selection should work:
- Selecting something on details opens it on notes
- What you do on notes page should not have any impact on select values on details page
- Regardless of what is selected all topics go into the final report

They (Brandon and Mallory) talked to the Casa folks that's the behavior they agreed on. The overall goal is just to not overwhelm volunteers with excessive text when they hit the notes page. (Source: https://rubyforgood.slack.com/archives/C0682C006L9/p1708372749226539 somewhere in that thread and we also talked over google meet)

Reply:
Gotcha. Definitely go with what the customer wants and Brandon and Mallory say.